browser-extension/config/const.js

```diff
@@ -15,7 +15,7 @@ const EXT_MODAL_VISIBLE_STATE = 'flex';
 const EXT_MODAL_INVISIBLE_STATE = 'none';
 // CSS selector identifying the element in the YT page the Tournesol modal
 // will wait for before being added to the DOM
-const EXT_MODAL_WAIT_FOR = 'div#info.ytd-watch-flexy';
+const EXT_MODAL_WAIT_FOR = 'div#content.ytd-app';
```

Comment:
Any reason to wait for this specific element ? 
Now that the iframe is displayed as a modal, maybe it could be appended to the body without waiting for this element ?

Reply:
I currently don't know, as I've always tested the modal with this behaviour.

I'm going to test.

Reply:
It seems to work without waiting for `div#content`.

I just removed the waiting statement.